articles/libraries/machine-learning/load.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Quantum machine learning library

```

Comment:
Add a description meta tag between title and author.
description: Learn how to load your own dataset to train a classifier model with the Microsoft QDK.